release/models/devices-manifest/openconfig-terminal-device-properties.yang

```diff
@@ -0,0 +1,501 @@
+module openconfig-terminal-device-properties {
+
+  // Yang version can be "1" without mandatory fields.
+  // Yang version 1.1 allows mandatory fiels in an augment
+  // https://tools.ietf.org/html/rfc7950#section-7.17
+  // If the augmentation adds mandatory nodes (see Section 3) that
+  // represent configuration to a target node in another module, the
+  // augmentation MUST be made conditional with a "when" statement.
+
+  yang-version "1";
+
+  // namespace
+  namespace "http://example.net/yang/openconfig-terminal-device-properties";
+  prefix "oc-opt-term-properties";
+
+  import openconfig-extensions { prefix oc-ext; }
+  import openconfig-transport-types { prefix oc-opt-types; }
+  import openconfig-terminal-device-property-types { prefix oc-opt-term-prop-types; }
+
+  // meta
+  organization "OpenConfig working group";
+
+  contact
+    "OpenConfig working group
+      www.openconfig.net";
+
+  description
+      "Module to extend OC terminal device with information about
+      operational modes. It supports operational modes for 1 Optical
+      Channel, with a single OTSi. The operational mode incluides key
+      attributes such modulation format, symbol rate, nominal central
+      frequency (NFC) tunability constraits (grid, min/max NCF), FEC
+      gain,minimum and maximum output power of the transmitter or
+      minimum q-value at the receiver as well as the spectrum width of
+      the OTSi (OTSiMC). It also includes (optional) aspects such as
+      filter characterization, CD and DGD tolerance.";
+
+  oc-ext:openconfig-version "0.1.0";
+
+
+  // Revisions
+  revision "2021-06-21" {
```

Comment:
we should probably update this before getting submitted?

Reply:
Updated to 2022-04-19